library/templates/dcla_culturalinstitutions.yml

```diff
@@ -0,0 +1,34 @@
+dataset:
+  name: &name dcla_culturalinstitutions
+  version: "{{ version }}"
+  acl: public-read
+  source:
+    url:
+      path: https://data.cityofnewyork.us/api/views/u35m-9t32/rows.csv
```

Comment:
The source paths that look like: `data.cityofnewyork.us/api/views/` actually indicate opendata sources. While this is the correct path to download from, we're going to transition to using the socrata API to get this data.

So instead of `url`, this should say `socrata`.

For example:
```yml
source:
    socrata:
      uid: u35m-9t32   
      format: csv
```